luvit-loader.lua

```diff
@@ -197,7 +197,7 @@ local function loader(dir, path, bundleOnly)
         return package.loaded[key]
       end
       local code = bundle.readfile(fullPath)
-      local module = loadstring(code, key)()
+      local module = loadstring(code, key)(key)
```

Comment:
Good idea, we should also include `@` in the path if it's a real path on the filesystem, then tools like luacov can work.